EventGenerator/src/SConscript

```diff
@@ -51,6 +51,7 @@ mainlib = helper.make_mainlib ( [
         'HepPDT',
         'HepPID',
         rootlibs,
+        'boost_filesystem',
```

Comment:
I do not see any uses of boost_filesystem in EventGenerators.  Why was the link dependency added?

Reply:
Whoops. I must have been playing around trying to solve a compilation error. Removed in commit a4ce17523